src/lib/helpers/fetchUser.ts

```diff
@@ -0,0 +1,7 @@
+import { getDocument } from '$sveltefire/firestore';
+import type { IUser } from '$lib/interfaces';
+
+export async function fetchUser(id: string): Promise<IUser> {
```

Comment:
This is function is not adding any benefit and is not needed. We can just make the database call right in the component.